### PR TITLE
Add sortable enums and update ts and ts-loader

### PR DESCRIPTION
**Pull request by Baza58**

### CODE REVIEW

**Comment by Baza58** [comment]
[Add enums for sorting and update TS and ts-loader in places JS](https://trello.com/c/5KR87yLg/344-add-enums-for-sorting-and-update-ts-and-ts-loader-in-places-js)

File: docs/places.md [hunk 1]
```diff
@@ -132,18 +132,18 @@ If you are on the second or latter page you can also call `prevPage` on the list
...
+Methods that support server side sorting take object with properties `sort` and `order` as a parameter. These methods' parameters interfaces extend [`Sortable`](https://github.com/Ceskasporitelna/cs-core-sdk-js/blob/master/lib/web-api/lists.ts) interface which takes enum as a generic. Both `sort` and `order` parameters are optional but you obviously cannot use `order` parameter without using `sort` parameter. Available `Sort` values are dependent on individual API and we provide you with these values in enums where possible. You can use these enums or just pass strings. `Order` can be `asc` for ascending which is also default or `desc` for descending. We provide [`Order`](https://github.com/Ceskasporitelna/cs-core-sdk-js/blob/master/lib/web-api/lists.ts#L156) enum for these values too.
 
 ```javascript
 
-    CSPlacesSDK
+    CSTransparentAccSDK
```
Comment (hauseja3) [inline docs/places.md]: Můžeš to prosim taky upravit pro `PlacesSDK` a příslušný sortable field?

File: docs/places.md [hunk 2]
```diff
@@ -155,17 +155,17 @@ You can use multiple values for sorting.
...
```
Comment (hauseja3) [inline docs/places.md]: Same here

**Comment by Baza58** [comment]
@hauseja3 Fixed.

**Comment by hauseja3** [comment]
@Baza58 Not really. Na places clientovi nemáme accounts. Prostě tam udělat example z places.

**Pull request merged**